apstra/api_iba_dashboards_test.go

```diff
@@ -114,13 +115,26 @@ func TestCreateReadUpdateDeleteIbaDashboards(t *testing.T) {
 			IbaWidgetGrid: [][]ObjectId{{widgetAId, widgetBId}, {widgetAId, widgetBId}},
 		}
 
-		t.Log("Test Create Dashboard")
+		ds, err := bpClient.GetAllIbaDashboards(ctx)
+		l := len(ds)
+		if len(ds) != 0 {
+			t.Fatalf("Expected no dashboards. got %d", l)
+		}
```

Comment:
Needs error check.

Variable `l` is set to `len(ds)`, and then `len(ds)` is called immediately after. Should choose between using `l` and `len(ds)`. I'd probably eliminate `l` and just invoke `len()` a second time in the problem case.

Reply:
un-checked error is above. It didn't quite make the context that GitHub has decided to share here in the conversation pane.

Reply:
Man! I was going to replace all lens with l. Got distracted before I finished the job :-)